### ISSUE TITLE
Possible newline issue in "workbench.action.terminal.runSelectedText"

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#13014]
Hello,

TL;DR: I'm wondering if LF and CRLF are incorrectly swapped in this ternary expression?

https://github-com/microsoft/vscode/blob/0899758daeebc6d5429679b4e9b5d76ee9794273/src/vs/workbench/contrib/terminal/browser/terminalActions.ts-L589

I've recently been experiencing some - when running Python code in the terminal with the "Run Selected Text In Active Terminal" keyboard shortcut. For example, - similar to [this](https://github-com/microsoft/vscode-python/-/8945) and [this](https://github-com/hoangKnLai/vscode-ipython/-/17).

While investigating, I noticed that the corresponding `workbench.action.terminal.runSelectedText` action chooses LF or CRLF, depending on the operating system (see link above).

My questions are:

1. Should LF and CRLF be swapped here? (so that Windows uses CRLF)
2. Would it be possible to make this `endOfLinePreference` value depend on a user-defined settings value, like `files.eol`?

My apologies if I'm completely on the wrong track with this!

Cheers!
